shared/src/styles.ts

```diff
@@ -47,6 +47,12 @@ export const BASE_FONT: types.CSSProperties = {
   fontSize: 'var(--jp-ui-font-size1, 13px)',
 };
 
+export const INPUT_TEXT_STYLE = {
```

Comment:
You could also use the spread syntax here.

```
export const INPUT_TEXT_STYLE = {
  ...BASE_FONT,
  color: 'var(--jp-ui-font-color1)',
};
```

Reply:
It really doesn't like the type conversion there, so it seems a lot simpler to keep it like this

Reply:
SGTM